diff --git a/open_spiel/games/CMakeLists.txt b/open_spiel/games/CMakeLists.txt
--- a/open_spiel/games/CMakeLists.txt
+++ b/open_spiel/games/CMakeLists.txt
@@ -99,6 +99,10 @@ set(GAME_SOURCES
   euchre/euchre.h
   first_sealed_auction/first_sealed_auction.cc
   first_sealed_auction/first_sealed_auction.h
+  fugitive/belief.cc
+  fugitive/belief.h
+  fugitive/fugitive.cc
+  fugitive/fugitive.h
   gin_rummy/gin_rummy.cc
   gin_rummy/gin_rummy.h
   gin_rummy/gin_rummy_utils.cc
@@ -524,6 +528,20 @@ add_executable(first_sealed_auction_test first_sealed_auction/first_sealed_aucti
                $<TARGET_OBJECTS:tests>)
 add_test(first_sealed_auction_test first_sealed_auction_test)
 
+add_executable(fugitive_test fugitive/fugitive_test.cc ${OPEN_SPIEL_OBJECTS}
+               $<TARGET_OBJECTS:tests>)
+add_test(fugitive_test fugitive_test)
+
+add_executable(fugitive_belief_test fugitive/belief_test.cc
+               ${OPEN_SPIEL_OBJECTS} $<TARGET_OBJECTS:tests>)
+add_test(fugitive_belief_test fugitive_belief_test)
+
+add_executable(fugitive_belief_experiment fugitive/belief_experiment.cc
+               ${OPEN_SPIEL_OBJECTS})
+
+add_executable(fugitive_baseline_experiment fugitive/baseline_experiment.cc
+               ${OPEN_SPIEL_OBJECTS})
+
 add_executable(garnet_test mfg/garnet_test.cc ${OPEN_SPIEL_OBJECTS}
                $<TARGET_OBJECTS:tests>)
 add_test(garnet_test garnet_test)
